src/services/TestDataService.ts

```diff
@@ -865,37 +867,23 @@ export class TestDataService {
     }
 
     /**
-     * Creates a system config entry
+     * Set system config for default sales channel
      *
-     * @param configurationKey - Config key for shop configurations.
-     * @param configurationValue - Config value as object for shop configurations (see {@link https://shopware.stoplight.io/docs/admin-api/9174d032146f8-create-a-new-system-config-resources|AdminApi Stoplight}).
-     * @param salesChannelId - Unique identity of sales channel.
+     * @param configs - Key value pairs to set
      */
-    async createSystemConfigEntry(
-        configurationKey: string,
-        configurationValue = {},
-        salesChannelId = '',
-    ): Promise<SystemConfig> {
-
-        const systemConfigStruct = {
-            id: this.IdProvider.getIdPair().uuid,
-            configurationKey: configurationKey,
-            configurationValue: configurationValue,
-            salesChannelId: salesChannelId || null,
-        }
-
-        const response = await this.AdminApiClient.post('system-config?_response=detail', {
-            data: systemConfigStruct,
+    async setSystemConfig(configs: Record<string, unknown>): Promise<void> {
+        const response = await this.AdminApiClient.post(`_action/system-config?_response=detail&salesChanneldId=${this.defaultSalesChannel.id}`, {
+            data: configs,
         });
         expect(response.ok()).toBeTruthy();
 
-        const { data: systemConfigEntry } = (await response.json()) as { data: SystemConfig };
-
-        this.addCreatedRecord('system_config', systemConfigEntry.id);
+        // we just unset all configs with this.defaultSalesChannel.id
+        // the defaults should take effect again
+        for (const key of Object.keys(configs)) {
+            this.restoreSystemConfig[key] = null;
```

Comment:
Where the restore is happening here? I just see that the values are set to zero but the database still have the newValues added by the method `setSystemConfig(configs)`. Or I interpreted the API Call wrong?

Reply:
https://github.com/shopware/acceptance-test-suite/pull/230/files#diff-a733b20ef0fec95da98ad2e7f38bb05601a59f1597f7fbfad202824c12dca4f5R1704-R1708